text/0000-promise.md

```diff
@@ -0,0 +1,460 @@
+- Feature Name: `JsPromise` and `TaskBuilder`
+- Start Date: 2020-09-17
+- RFC PR: (leave this empty)
+- Neon Issue: (leave this empty)
+
+## Summary
+[summary]: #summary
+
+Provides an ergonomic API for executing tasks on the libuv threadpool and notifying JavaScript on the main thread with a result.
```

Comment:
This wording caught me off guard at first—I think you're saying the libuv thread pool is specifically the `TaskBuilder` half of the RFC, not the `JsPromise` half, right?

Reply:
That's correct. I think it will be more clear if I break it into two sentences since the next line explains why they are being designed together.